util/messages/validateTransactionSuccess.ts

```diff
@@ -0,0 +1,11 @@
+import { DeliverTxResponse, isDeliverTxFailure } from '@cosmjs/stargate'
+
+export function validateTransactionSuccess(result: DeliverTxResponse) {
```

Comment:
nice utility function

Reply:
so weird the transaction itself wouldn't error out 🤷